compiler/rustc_resolve/src/lib.rs

```diff
@@ -422,7 +422,9 @@ enum ModuleKind {
     ///
     /// This could be:
     ///
-    /// * A normal module ‒ either `mod from_file;` or `mod from_block { }`.
+    /// * A normal module ‒ either `mod from_file;` or `mod from_block { }` —
+    ///   or the crate root (which is conceptually a top-level module).
+    ///   Note that the crate root's [name][Self::name] will be [`kw::Invalid`].
```

Comment:
Is this correct? I think that's what I saw in the code, but I may have been mistaken since I don't know the code.

Reply:
Yes, this is correct.